tap_mixpanel/__init__.py

```diff
@@ -45,7 +45,14 @@ def main():
         LOGGER.warning("start_date greater than 1 year maxiumum for API.")
         LOGGER.warning("Setting start_date to 1 year ago, %s", start_date)
 
+    #Check support for EU endpoints
+    if str(parsed_args.config.get('eu_residency_server')).lower() == "true":
```

Comment:
@prijendev Please add unit test cases for this code change

Reply:
Added unit test cases for the same.